### PR TITLE
Fixing typos

### PR DESCRIPTION
**Pull request by DianaBellefeuille**
Fixed spacing.

### CODE REVIEW

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -30,18 +30,19 @@
...
-     private final Hood mHood = Hood.create();
-     private final Hopper mHopper = Hopper.create();
-     private final Indexer mIndexer = Indexer.create();
-     private final Intake mIntake = Intake.create();
+     private final Drivetrain mDrivetrain =       Drivetrain.create();
+     private final Flywheel mFlywheel =           Flywheel.create();
+     private final Hood mHood =                   Hood.create();
+     private final Hopper mHopper =               Hopper.create();
+     private final Indexer mIndexer =             Indexer.create();
+     private final Intake mIntake =               Intake.create();
```
Comment (ghost) [inline src/main/java/frc/robot/RobotContainer.java]: Hmm....I don't like these changes.

**Review by ghost — CHANGES_REQUESTED** [review]
See my comments

**Review by DianaBellefeuille — COMMENTED** [review]
Unfixed typos according to erics complaint

File: src/main/java/frc/robot/RobotContainer.java (hunk 1 above)
Comment (DianaBellefeuille) [reply src/main/java/frc/robot/RobotContainer.java]: unfixed

**Review by ghost — APPROVED** [review]
GRRRRRREAT!!!!

**Review by 22nadsup — APPROVED** [review]
Solid

**Pull request merged**